src/index.js

```diff
@@ -31,6 +32,22 @@ import 'index.css';
 
 import App from 'App';
 
+Sentry.init({
+  dsn: 'https://dac0c436847fc922e68de0855b5ff738@o4506299756642304.ingest.sentry.io/4506345074130944',
```

Comment:
Question: Should this be defined through a environment variable? I don't think this should be used by anyone using the repo